src/main/java/aquality/selenium/core/configurations/ILoggerConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package aquality.selenium.core.configurations;
+
+/**
+ * Describes logger configuration.
+ */
+public interface ILoggerConfiguration {
+
+    /**
+     * Gets language of the library.
```

Comment:
maybe 'of the logger' instead of 'of the library.'?

Reply:
fixed